third_party/terraform/website/docs/r/dns_record_set.markdown

```diff
@@ -163,10 +163,11 @@ Only the arguments listed above are exposed as attributes.
 
 ## Import
 
-DNS record set can be imported using the `zone name`, `record name` and record `type`, e.g.
+DNS record set can be imported using the `zone name`, `record name` and record `type`, with an optional `project`, e.g.
 
 ```
 $ terraform import google_dns_record_set.frontend prod-zone/frontend.prod.mydomain.com./A
+$ terraform import google_dns_record_set.frontend prod-project/prod-zone/frontend.prod.mydomain.com./A
```

Comment:
We use the `{{project}}/{{zone}}/` syntax in most other resources. We should probably update this to be consistent.